internal/driver/html/graph.css

```diff
@@ -0,0 +1,7 @@
+#graph {
```

Comment:
How about just including this in the existing css file? Not sure a new tiny file is worth having.

Reply:
This will make `common.css` not really "common" (one-off style that is applied to a single page). Do you prefer to have this style in the common css file?

Reply:
Makes sense.